src/Altinn.Common/Altinn.Common.PEP/Altinn.Common.PEP/Clients/IHttpClientAccessor.cs

```diff
@@ -0,0 +1,35 @@
+using System.Net.Http;
+
+namespace Altinn.Common.PEP.Clients
+{
+    /// <summary>
+    /// Interface for http client accessor
+    /// </summary>
+    public interface IHttpClientAccessor
```

Comment:
As you only need the Authz client I would make a single accessor for that client removing all references to remaining platform components.